dotnetConsole/.conf/install-deps-scripts/install-dotnet-sdk-8.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+echo "bbbbb"
```

Comment:
![bunnies-what](https://github.com/toradex/vscode-torizon-templates/assets/2633321/6e76a036-8ad6-4bb6-a203-557ee0a0daec)

Reply:
Ops, residual echo from testing the update script, I will remove it here